applications/luci-app-usteer/root/usr/share/rpcd/acl.d/luci-app-usteer.json

```
@@ -6,6 +6,7 @@
         "uci": [ "get" ],
         "usteer": [ "remote_info","remote_hosts", "local_info", "get_clients"],
         "luci-rpc": [ "getHostHints" ]
+        "network.rrdns": [ "lookup" ]
```

I think this needs a comma:
```suggestion
        "luci-rpc": [ "getHostHints" ],
        "network.rrdns": [ "lookup" ]
```

---

i think so too :)